### PR TITLE
Change order for joining subnets

### PR DESCRIPTION
**Pull request by sukantoraymond**
# Docs PR Template

## Why this should be merged

Changing the order of adding validator so that users will call join before calling addValidator to minimize occurrences of Subnet downtime.

## How this works

<please give a thorough description of what your changes are and how they work>

## How these changes were tested 

<please give a thorough description of how to view the changes and how they were tested>

## Workflow checks

- [x] ran `vale /docs/file/path` on all changed `.md` files to ensure all grammar rules pass
- [x] ran `markdownlint /docs/file/path` on all changed `.md` files to ensure all linting rules pass
- [x] completed the above two checks and all additional rules outlined in `style-checker-notes.md` to
  ensure all checks pass
- [x] if a doc was removed/deleted, the path to that doc must be redirected to a valid URL via the
  `_redirects` file
- [x] all files that were moved from their current directory to a new path have had their paths
  redirected via the `_redirects` file
- [x] `_redirects` were manually verified with the cloudflare preview link
- [x] `sidebars.json` reflects all changes made to file path

### CODE REVIEW

File: docs/subnets/create-a-fuji-subnet.md [hunk 1]
```diff
@@ -710,7 +587,12 @@ Use the arrow keys to navigate: ↓ ↑ → ←
 
 The default is `Yes` but just choose `No` here to speed up things, assuming the node is already whitelisted.
 
-There are now two choices possible: automatic and Manual configuration. As mentioned earlier,
+As mentioned earlier, If the node has not been whitelisted, a holder of the Subnet control key
```
Comment (dasconnor) [inline docs/subnets/create-a-fuji-subnet.md:590]: We want to change what we suggest here, we should be advising that addValidator always be called AFTER join. Also capitalization of If.

File: docs/subnets/create-a-fuji-subnet.md [hunk 2]
```diff
@@ -710,7 +587,12 @@ Use the arrow keys to navigate: ↓ ↑ → ←
 
 The default is `Yes` but just choose `No` here to speed up things, assuming the node is already whitelisted.
 
-There are now two choices possible: automatic and Manual configuration. As mentioned earlier,
+As mentioned earlier, If the node has not been whitelisted, a holder of the Subnet control key 
+_must_ call
+[Subnet addValidator](../apis/avalanchego/apis/p-chain.md#platformaddsubnetvalidator) defined in 
+the next section in order to allow the node to validate the Subnet.
+
+Next, there are two setup choices: automatic and Manual configuration. As mentioned earlier,
```
Comment (dasconnor) [inline docs/subnets/create-a-fuji-subnet.md:595]: case

File: docs/subnets/create-a-fuji-subnet.md [hunk 3]
```diff
@@ -803,6 +685,125 @@ After you update your config, you are going to need to restart your node for the
 take effect.
 ```
 
+## Add a Validator
+
+:::warning
+
+If the `join` command isn't successfully completed before `addValidator` is completed and the
```
Comment (dasconnor) [inline docs/subnets/create-a-fuji-subnet.md:692]: Remove the `>20%` part. Just say the subnet may have degraded performance or even halt. Be scary haha.

**Review by dasconnor — COMMENTED** [review]
nice job

File: docs/subnets/create-a-fuji-subnet.md [hunk 4]
```diff
@@ -690,13 +572,8 @@ Use the arrow keys to navigate: ↓ ↑ → ←
...
 ```
 
-The [Deploy the Subnet](#deploy-the-subnet) section shows that a Subnet is permissioned
-via a set of keys. Therefore not any node is suitable as validator to the Subnet. A holder of a control
-key _must_ call
-[Subnet addValidator](../apis/avalanchego/apis/p-chain.md#platformaddsubnetvalidator) first in order
-to allow the node to validate the Subnet. So the tool allows the user now to verify if the node has
-already been permissioned -"whitelisted"- to be a validator for this Subnet -by calling an API in
-the background-.
+Next, you will be asked if you want to verify if your node has already been permissioned
```
[the remark this answers is not in the document]
Comment (dasconnor) [reply docs/subnets/create-a-fuji-subnet.md]: did you create a ticket for this or should I?

File: docs/subnets/create-a-fuji-subnet.md [hunk 5]
```diff
@@ -689,11 +672,14 @@ take effect.
 
 :::warning
 
-If the `join` command isn't successfully completed before `addValidator` is completed and the 
-validator's stake weight is >20% of the Subnet total, the Subnet may have down time.
+If the `join` command isn't successfully completed before `addValidator` is completed, the Subnet 
+may experience degraded performance.
```
Comment (dasconnor) [inline docs/subnets/create-a-fuji-subnet.md:676]: could experience degraded performance or even halt.

File: docs/subnets/create-a-fuji-subnet.md [hunk 6]
```diff
@@ -689,11 +672,14 @@ take effect.
...
 
 :::
 
+Now that the node has joined your Subnet, if you are a holder of the Subnet control key, you
```
Comment (dasconnor) [inline docs/subnets/create-a-fuji-subnet.md:680]: Now that the node has joined the Subnet, a Subnet control key holder must call `addValidator`

File: docs/subnets/create-a-fuji-subnet.md [hunk 7]
```diff
@@ -541,29 +541,154 @@ To get your new Subnet information, visit the
...
 
 See [here](../nodes/validate/add-a-validator.md#add-a-validator-with-avalanche-wallet) on how to
 become a validator.
 
 :::
 
-This new Subnet is cool - but it doesn't have any dedicated validators yet. Add one by running
-the `addValidator` command and adding the name of the Subnet. To be clear, this does _not start or
-run_ a validator, it only whitelists the node as a recognized validator on the Subnet.
+To add a node as a validator to a Subnet, run the `join` command along with the Subnet name:
```
Comment (meaghanfitzgerald) [inline docs/subnets/create-a-fuji-subnet.md:559]:
```suggestion
First, request permission to validate by running the `join` command along with the Subnet name:
```

File: docs/subnets/create-a-fuji-subnet.md [hunk 8]
```diff
@@ -541,29 +541,154 @@ To get your new Subnet information, visit the
 search best works by blockchain ID, so in this example, enter `2XDnKyAEr1RhhWpTpMXqrjeejN23vETmDykVzkb4PrU1fQjewh`
 into the search box and you should see your shiny new blockchain information.
 
-## Add a Validator
+## Join a Subnet
```
Comment (meaghanfitzgerald) [inline docs/subnets/create-a-fuji-subnet.md:544]:
```suggestion
## Request to Join a Subnet as a Validator
```

File: docs/subnets/create-a-fuji-subnet.md [hunk 9]
```diff
@@ -541,29 +541,154 @@ To get your new Subnet information, visit the
...
+
+This new Subnet is cool - but it doesn't have any dedicated validators yet. We are now going to add 
+a validator to the Subnet.
```
Comment (meaghanfitzgerald) [inline docs/subnets/create-a-fuji-subnet.md:547]:
```suggestion
The new Subnet created in the previous steps doesn't have any dedicated validators yet. 
To request permission to validate a Subnet, the following steps are required:
```

**Comment by meaghanfitzgerald** [comment]
A few nits to clear up the language, then LGTM

File: docs/subnets/create-a-mainnet-subnet.md [hunk 10]
```diff
@@ -249,7 +249,10 @@ appears:
 
 Well done. You have just created your own Subnet running on `Mainnet`. Now it's time to add your validators.
 
-## Add a Validator
+## Join a Subnet
+
+This new Subnet is cool - but it doesn't have any dedicated validators yet. We are now going to add
+a validator to the Subnet.
```
Comment (meaghanfitzgerald) [inline docs/subnets/create-a-mainnet-subnet.md:255]:
```suggestion
## Request to Join a Subnet as a Validator

The new Subnet created in the previous steps doesn't have any dedicated validators yet. 
To request permission to validate a Subnet, the following steps are required:
```

consistency with the fuji tutorial

File: docs/subnets/create-a-mainnet-subnet.md [hunk 11]
```diff
@@ -261,26 +264,153 @@ a validator.
 
 :::
 
-This new Subnet is cool, but it doesn't have any dedicated validators and can't process transactions
-yet.
+To add a node as a validator to a Subnet, run the `join` command along with the Subnet name:
```
Comment (meaghanfitzgerald) [inline docs/subnets/create-a-mainnet-subnet.md:267]:
```suggestion
First, request permission to validate by running the `join` command along with the Subnet name:
```

consistent with the fuji tutorial

**Review by meaghanfitzgerald — APPROVED** [review]

**Pull request merged**